src/base/gfx/pamwriter.cpp

```diff
@@ -7,8 +7,8 @@ PamWriter::PamWriter(std::ostream &stream) : stream(stream) {}
 void PamWriter::write(const PixMapView &pixmap)
 {
 	stream << "P7\n"
-	       << "WIDTH " << (int)pixmap.size.x << "\n"
-	       << "HEIGHT " << (int)pixmap.size.y << "\n"
+	       << "WIDTH " << static_cast<int>(pixmap.size.x) << "\n"
```

Comment:
PamReader and PamWriter are not used, the whole file can be removed.

Reply:
OK